homeassistant/components/publibike/config_flow.py

```diff
@@ -0,0 +1,83 @@
+"""Config flow for the PubliBike Public API integration."""
+
+from __future__ import annotations
+
+import logging
+
+from pypublibike.publibike import PubliBike
+from requests import RequestException
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.helpers import config_validation as cv
+
+from .const import (
+    BATTERY_LIMIT,
+    BATTERY_LIMIT_DEFAULT,
+    DOMAIN,
+    LATITUDE,
+    LONGITUDE,
+    STATION_ID,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(BATTERY_LIMIT, default=BATTERY_LIMIT_DEFAULT): vol.All(
+            cv.positive_int, vol.Range(1, 100)
+        ),
+        vol.Optional(STATION_ID): cv.positive_int,
+        vol.Optional(LATITUDE): cv.latitude,
+        vol.Optional(LONGITUDE): cv.longitude,
```

Comment:
What is a user enters both station ID and lat/long?

Reply:
Station ID would have a priority and location would be ignored. Should this also be made explicit in the schema here?